addons/binding/org.openhab.binding.nest/src/main/java/org/openhab/binding/nest/handler/NestBridgeHandler.java

```diff
@@ -0,0 +1,268 @@
+package org.openhab.binding.nest.handler;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.commons.httpclient.URIException;
+import org.apache.commons.httpclient.util.URIUtil;
+import org.eclipse.smarthome.config.core.Configuration;
+import org.eclipse.smarthome.core.thing.Bridge;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.binding.BaseBridgeHandler;
+import org.eclipse.smarthome.core.thing.binding.ThingHandler;
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.core.types.RefreshType;
+import org.eclipse.smarthome.io.net.http.HttpUtil;
+import org.openhab.binding.nest.NestBindingConstants;
+import org.openhab.binding.nest.config.NestBridgeConfiguration;
+import org.openhab.binding.nest.discovery.NestDiscoveryService;
+import org.openhab.binding.nest.internal.NestAccessToken;
+import org.openhab.binding.nest.internal.NestDeviceAddedListener;
+import org.openhab.binding.nest.internal.NestUpdateRequest;
+import org.openhab.binding.nest.internal.data.Camera;
+import org.openhab.binding.nest.internal.data.NestDevices;
+import org.openhab.binding.nest.internal.data.Structure;
+import org.openhab.binding.nest.internal.data.Thermostat;
+import org.openhab.binding.nest.internal.data.TopLevelData;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+public class NestBridgeHandler extends BaseBridgeHandler {
```

Comment:
Please add a JavaDoc with `@author` tag containing a real name of the author. See [Code Style](http://docs.openhab.org/developers/development/guidelines.html#a-code-style).